chapters/ch15.md

```diff
@@ -9,30 +9,36 @@ Nhóm của bạn có một số ý tưởng cải thiện ứng dụng nhận d
 
 * Fix the problem of your algorithm recognizing ​dogs​ as cats.
 ->
+
 Sửa lỗi nhận diện chó thành mèo trong thuật toán.
 
 * Fix the problem of your algorithm recognizing ​great cats​ (lions, panthers, etc.) as house cats (pets).
 ->
+
 Sửa lỗi nhận diện thú họ mèo (sư tử, báo, v.v) thành mèo nhà (thú nuôi).
 
 
 * Improve the system’s performance on ​blurry​ images.
 ->
+
 Cải thiện hiệu quả của hệ thống trên dữ liệu ảnh mờ.
 
 
 * ...
 ->
+
 ...
 
 You can efficiently evaluate all of these ideas in parallel. I usually create a spreadsheet and fill it out while looking through ~100 misclassified dev set images. I also jot down comments that might help me remember specific examples. To illustrate this process, let’s look at a spreadsheet you might produce with a small dev set of four examples:
 ->
+
 Bạn có thể đánh giá song song tất cả các ý kiến trên một cách có hiệu quả. Tôi thường tạo một bảng biểu và điền vào đó khi phân tích tập phát triển với khoảng 100 ảnh bị phân loại nhầm. Tôi cũng viết xuống các ghi chú có thể giúp tôi nhớ được những trường hợp đặc biệt. Để minh họa cho quá trình này, bạn có thể tham khảo bảng biểu được tạo ra từ một tập phát triển nhỏ với bốn mẫu dưới đây:       
```

Comment:
hình như (1) "khi phân tích tập phát triển với khoảng 100 ảnh bị phân loại nhầm" nên là (2) " khi phân tích 100 ảnh bị phân loại nhầm ở trong tập phát triển". Nhìn qua thì 2 câu này giống nhau; nhưng thực tết lại khác: (1) refers đến việc phân tích toàn bộ dev set & trong đó có 100 ảnh nhầm còn (2) refers đến việc collect 100 ảnh bị gán nhãn nhầm  trong dev set, sau đó phân tích 100 ảnh này.